.github/workflows/create_release.yml

```
@@ -36,3 +36,27 @@ jobs:
           # assignees:
           body: |
             Releasing ${{ steps.version.outputs.next_tag }}.
+      - name: Check out delphi epidata 
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.EPIDATA_PAT }}
+          repository: cmu-delphi/delphi-epidata
+          path: delphi-epidata
+      - name: Copy files
+        run: |
+          ls -al
+          cp src/stores/descriptions.raw.txt delphi-epidata/src/server/endpoints/covidcast_utils/descriptions.raw.txt
+          cp src/stores/questions.raw.txt delphi-epidata/src/server/endpoints/covidcast_utils/questions.raw.txt
```

less verbosity for clarity
```suggestion
          cp src/stores/descriptions.raw.txt src/stores/questions.raw.txt \
             delphi-epidata/src/server/endpoints/covidcast_utils/
```